css/pages/card/card.html

```diff
@@ -0,0 +1,283 @@
+                                <div class="primary_headaer">
+                                    Valorant Gaming
+                                </div>
+                                <div class="secondary_headaer">
+                                    By Swapnil Bansal
+                                </div>
+                            </div>
+                            <div class="card_content_body">
+                                Leh is the joint capital and largest town of the union territory of Ladakh in India.
+                                Leh, located in the
+                                Leh district, was also the historical capital of the Kingdom of Ladakh,
+                            </div>
+                        </div>
+                        <div class="card_actions">
+                            <div class="card_button_action">
+                                <button class="btn_card">READ</button>
+                                <button class="btn_card">BOOKMARK</button>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div>
+                    <iframe src="https://www.thiscodeworks.com/embed/61fa9d71b783be0015bbaee0"
+                        style="width: 100%; height: 398px;" frameborder="0">
+                    </iframe>
+                </div>
+            </div>
+        </div>
+        <div id="card_only_icons">
+            <h3 class="main_container_heading fs-m py-x-small">Card with Icons Only</h3>
+            <hr>
+            <p class="main_container_description py-x-small">This one is also a variant of the basic card from example 1
+                with a slight variation i.e it does not
+                have any buttons.</p>
+            <div>
+                <div class="preview_container p-x-small">
+                    <div class="card card_onlyIcon">
+                        <div class="card_content">
+                            <img class="card_content_image" src="../../vendors/card_image.png" , alt="">
+                            <div class="card_content_header">
+                                <div class="primary_header">
+                                    Valorant Gaming
+                                </div>
+                                <div class="secondary_header">
+                                    By Swapnil Bansal
+                                </div>
+                            </div>
+                            <div class="card_content_body">
+                                Leh is the joint capital and largest town of the union territory of Ladakh in India.
+                                Leh, located in the
+                                Leh district, was also the historical capital of the Kingdom of Ladakh,
+                            </div>
+                        </div>
+                        <div class="card_actions card_onlyIcon">
```

Comment:
be consistent with naming of classes i.e onlyIcon can be changed to only-icon

Reply:
Thanks removed the class.Not required